pytest_rerunfailures.py

```diff
@@ -34,6 +35,14 @@ def pytest_addoption(parser):
         type=int,
         default=0,
         help="number of times to re-run failed tests. defaults to 0.")
+    group._addoption(
+        '--reruns-delay',
+        action='store',
+        dest='reruns_delay',
+        type=float,
+        default=0,
+        help='add time (seconds) delay between reruns.'
+    )
```

Comment:
In addition to the addition to the command line it is also helpful to have `reruns_delay` added in the documentation for the config. https://github.com/pytest-dev/pytest-rerunfailures/pull/55/files#diff-c3a581865b7c0be4e6b5d5fa60d6aed5R51